internal/nostr/nostr.go

```diff
@@ -647,7 +647,7 @@ func (svc *Service) StopSubscriptionHandler(c echo.Context) error {
 
 	svc.Logger.WithFields(logrus.Fields{
 		"subscription_id": subscription.ID,
-	}).Info("Stopped subscription")
+	}).Debug("Stopped subscription")
```

Comment:
Changing this back to Info as this is not used for /nip47 where subscription ID is 0 (this is the stop handler which is used for normal subscriptions)

Reply:
what does that mean?

Reply:
I thought you changed this to Debug because we use subscriptions for /nip47 handler too and it might be getting triggered often as we stop the subscriptions after processing a single /nip47 request

But this is logged in the stop subscription handler which is solely for "actual" subscriptions and not /nip47 requests and hence is not logged frequently (currently it is not even logged at all)